.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -4,6 +4,14 @@ about: Submit a bug report to help us improve Habitat
 
 ---
 
+## Habitat-Sim version
+vX or master?
+
+Habitat is under active development, and we advise users to restrict themselves to stable releases. Are you using the latest [release version](https://github.com/facebookresearch/habitat-sim/releases) of Habitat-Sim? If not, why not?
+(We may not be able to help with problems in earlier versions.)
+
+Master branch contains 'bleeding edge' code and should be used at your own risk.
```

Comment:
Maybe worth to make it softer as we are still interested in bug reports for master branch.